### ISSUE TITLE
IFTTT Maker

### ISSUE THREAD

**Issue by adanchenko** [mcolyer/hacklet-remote#4]
Is it possible to use IFTTT's Maker channel? It provides a webhook.

**Comment by mcolyer** [comment]
@adanchenko yes [this channel](https://ifttt.com/maker) is vastly superior than the hack that I was using before. Thanks for mentioning it, would you be interested in making the modifications to support it?

**Comment by mcolyer** [comment]
Closing in favor of #5.
